spec/controllers/manage_assessments/attachments_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+describe ManageAssessments::AttachmentsController do
+  describe "GET #show" do
+    it "downloads the requested attachment from server" do
+      course = create(:course)
+      outcome = create(:outcome)
+      coverage = create(:coverage, course: course, outcomes: [outcome])
+      outcome_coverage = OutcomeCoverage.first
+      assignment = create(:assignment, outcome_coverage: outcome_coverage)
+      attachment = Attachment.create(file: File.open("spec/fixtures/attachments/example.pdf"), attachable: assignment)
```

Comment:
I do think this would make for a useful factory. Then you could `create(:attachment, attachable: assignment)` and we could default to using the example pdf.